metadata-ingestion/tests/integration/superset/test_superset.py

```diff
@@ -306,3 +313,41 @@ def test_superset_stateful_ingest(
             output_path=deleted_mces_path,
             golden_path=test_resources_dir / "golden_test_stateful_ingest.json",
         )
+
+
+@freeze_time(FROZEN_TIME)
+@pytest.mark.integration
+def test_preset_ingest(pytestconfig, tmp_path, mock_time, requests_mock):
+    test_resources_dir = pytestconfig.rootpath / "tests/integration/superset"
+
+    register_mock_api(request_mock=requests_mock)
+
+    pipeline = Pipeline.create(
+        {
+            "run_id": "preset-test",
+            "source": {
+                "type": "preset",
+                "config": {
+                    "connect_uri": "mock://mock-domain.superset.com/",
+                    "api_key": "KEY",
+                    "api_secret": "SECRET",
+                },
+            },
+            "sink": {
+                "type": "file",
+                "config": {
+                    "filename": f"{tmp_path}/preset_mces.json",
+                },
+            },
+        }
+    )
+
+    pipeline.run()
+    pipeline.raise_from_status()
+    golden_file = "golden_test_ingest.json"
```

Comment:
You'll need separate preset golden file as it will have platform preset in entity urns.

Reply:
This can help you generate golden file from scratch - https://datahubproject.io/docs/metadata-ingestion/developing/#updating-golden-test-files